src/components/Modals/SafetyAlertModal/SafetyAlertModal.tsx

```diff
@@ -0,0 +1,95 @@
+import { useEffect, useState } from 'react';
+import moment from 'moment';
+import { useLocalStorage } from 'usehooks-ts';
+import { api } from '@/hooks';
+import { DerivLightIcWarningIcon } from '@deriv/quill-icons';
+import { Localize } from '@deriv-com/translations';
+import { Button, Modal, Text } from '@deriv-com/ui';
+import './SafetyAlertModal.scss';
+
+const SafetyAlertModal = () => {
+    const { data } = api.account.useActiveAccount();
+    const [isModalOpen, setIsModalOpen] = useState(false);
```

Comment:
the reason why im handling this internally in the modal is because when I use the traditional method of useModalManager `showModal`, it will briefly appear because for a brief moment, the localstorage function to grab it is quite slow, so even though there may be a timestamp there, it will default it to an empty string, so it will briefly show the modal. And theres so many useEffects in the app content file, so it will add it to the DOM, then remove it, then add it, which makes it look buggy.

Handling it this way, it won't ever get affected by the useEffects at the top level, and it is handled internally